configuration/messageproperties/messages_zl_fr.properties

```diff
@@ -105,4 +105,10 @@ pih.app.hiv.fingerprint.status.title=Statut des empreintes digitales
 zl.fingerprint.collected=Empreinte digitale collectée
 zl.fingerprint.not.collected=Empreinte digitale non collectée
 zl.collected.on=Collecté le:
-zl.fingerprint-status=Statut:
\ No newline at end of file
+zl.fingerprint-status=Statut:
+mch.dashboard.prenatalVisits=VISITES PRENATALES
+zl.status.mch.numberOfPrenatalVisits=Nombre de visites prénatales
+zl.status.mch.expectedDeliveryDate=Date probable d'accouchement
+zl.status.mch.lastPrenatalVisitDate=Date de la dernière visite prénatale
+zl.status.mch.expectedDeliveryDate.none=Aucune 
```

Comment:
You don't need to create duplicate messages.  You can use an existing one and share.  These already exists:

pihcore.pregnancy.dueDate=Date probable d'accouchement (DPA)
pihcore.none.label=aucun(e)

Same for the English file.